### PR TITLE
Tour of Beam markdown touchups

### PR DESCRIPTION
**Pull request by hjtran**
Just some formatting and clarification changes to a few Tour of Beam pages. (I've been referring many people to Tour of Beam!)
  - Broke up the first page of the Tour of Beam which was just a huge blob of unbroken text.
 Before: 
image
After: (just with local markdown rendering)
image

- Adjusted `PTransform` definition which used to say that PTransforms took one or more PCollection when they can really take zero or more.

- Small capitalization / language clarifications

### CODE REVIEW

**Comment by hjtran** [comment]
assign to next reviewer

**Comment by lostluck** [comment]
Sorry about the delay. The tail end of Google internal release tasks made me miss this earlier this week. Looking now.

File: learning/tour-of-beam/learning-content/introduction/introduction-concepts/pipeline-concepts/overview-pipeline/description.md [hunk 1]
```diff
@@ -22,7 +22,7 @@ The Beam SDKs provide several abstractions that simplify the mechanics of large-
 
 → `PCollection`: A PCollection represents a distributed data set that your Beam pipeline operates on. The data set can be bounded, meaning it comes from a fixed source like a file, or unbounded, meaning it comes from a continuously updating source via a subscription or other mechanism. Your pipeline typically creates an initial PCollection by reading data from an external data source, but you can also create a PCollection from in-memory data within your driver program. From there, PCollections are the inputs and outputs for each step in your pipeline.
 
-→ `PTransform`: A PTransform represents a data processing operation, or a step, in your pipeline. Every PTransform takes one or more PCollection objects as the input, performs a processing function that you provide on the elements of that PCollection, and then produces zero or more output PCollection objects.
+→ `PTransform`: A PTransform represents a data processing operation, or a step, in your pipeline. Every PTransform takes zero or more PCollection objects as the input, performs a processing function that you provide on the elements of that PCollection, and then produces zero or more output PCollection objects.
```
Comment (lostluck) [inline learning/tour-of-beam/learning-content/introduction/introduction-concepts/pipeline-concepts/overview-pipeline/description.md:25]: I'm on the fence between "yes, this is accurate and technically correct" and "no, this doesn't help users learn the model, as it's easier to automatically follow best practices by treating the 0 input cases as special/exceptional".

But I don't feel strongly enough for the latter to force further rewrites.

File: learning/tour-of-beam/learning-content/introduction/introduction-concepts/runner-concepts/description.md [hunk 2]
```diff
@@ -61,9 +61,9 @@ In java, you need to set runner to `args` when you start the program.
 {{end}}
 
 {{if (eq .Sdk "python")}}
-In the Python SDK , the default is runner **DirectRunner**.
+In the Python SDK , the **DirectRunner** is the default runner and is used if no runner is specified.
```
Comment (lostluck) [inline learning/tour-of-beam/learning-content/introduction/introduction-concepts/runner-concepts/description.md:64]: No action required.

Obligatory complaint that we never explain anywhere that the Direct Runner isn't a monolith and has very different behaviors between SDKs. I can't finish Prism soon enough...

**Review by lostluck — APPROVED** [review]
Approved! Nothing blocking so I'll merge, but we can always discuss and add another PR later.  

Sorry again for the delay.

**Pull request merged**